core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -11,6 +11,7 @@
     String searchFieldDefaultValue = "store_id";
     String searchFieldValueDefaultValue = "2";
     String servicePathCartFetchItems = "/us/V1/guest-carts/";
+    String recaptchaSiteKey = "6LfrFKQUAAAAAMzFobDZ7ZWy982lDxeps8cd1I2i";
```

Comment:
@nitheshkumar-hcl  This is magento service config, Create separate common configuration

Reply:
Committed the changes as per the review comments